packages/legend-query/src/components/QueryBuilderValueSpecificationEditor.tsx

```diff
@@ -14,7 +14,13 @@
  * limitations under the License.
```

Comment:
I will review this file last. Please update the video to the latest so we can make some discussion on styling as well. Thanks!